cheatsheets/Error_Handling_Cheat_Sheet.md

```diff
@@ -107,7 +109,7 @@ response.setStatus(200);
 {"message":"An error occur, please retry"}
 ```
 
-## Java SpringMVC/SpringBoot web application
+### Java SpringMVC/SpringBoot web application
```

Comment:
Can this be made more generic so drop the slash in the middle and simplify linking?